### ISSUE TITLE
Feature Request: add option to subscribe to Pull Request review requests for a specific individual / team

### ISSUE THREAD

**Issue by jeherve** [integrations/slack#1434]
**Is your feature request related to a problem? Please describe.**

The `/github subscribe` command is a great start to start integrating GitHub into our team's Slack channel. However, it does not include any easy way to get notifications whenever someone requests a review from our GitHub team on a Pull Request in a repository of our organization.

**Describe the solution you'd like**

Ideally, I would love to see a new option such as:

`/github subscribe owner/repo review_requested [team_name]

**Describe alternatives you've considered**

I've considered relying on [scheduled reminders](https://docs.github.com/en/organizations/organizing-members-into-teams/managing-scheduled-reminders-for-your-team) for this, but they lack some things that I believe the Slack integration may be able to offer:

- Be notified once only.
- Be notified as soon as the review is requested.
- Be notified only for review requests for the team (scheduled reminders gather a list of Pull Requests pending review from the team **and** from individual members on the team)

**Additional context**

Related issue:
- #382

**Comment by drewtwitchell** [comment]
Big fan of this! I hope it comes to the GitHub slack integration soon!

**Comment by istisiki** [comment]
+1 I would love to have this option.

**Comment by pkukulski** [comment]
I would also love to see this implemented or at least being able to filter by specific label

**Comment by GalGoldnerVizAI** [comment]
+1

**Comment by CodesOf-Partha** [comment]
+1

**Comment by wlai-wavefin** [comment]
+1

**Comment by Kirk-Patrick** [comment]
Looking forward to this also +1.

**Comment by SauravPahadia** [comment]
+1

**Comment by francescopepe** [comment]
+1

**Comment by AndreiaMatiasSilva** [comment]
+1

**Comment by jjkazm** [comment]
same here, cant wait for this future

**Comment by aleksnamaniuk** [comment]
Great feature, waiting for it super badly :)

**Comment by amanda-allison** [comment]
+1 would love to have this option!

**Comment by nwbt** [comment]
this would save lots of time

**Comment by riyo-tiket** [comment]
would love to have this, 
It's would also nice to have notification when our PR were commented.

**Comment by georgi-denchew** [comment]
+1

**Comment by jbart12** [comment]
+1 please

**Comment by srequiem** [comment]
+1 for reeaaaaal

**Comment by jbart12** [comment]
this works for me until there is an official integration:
https://rasim.pro/blog/personal-slack-github-notifications-for-pull-requests-reviewers-how-to-set-up/

**Comment by emma-nayya** [comment]
+1 looking for this exact feature

**Comment by jeherve** [comment]
> this works for me: [rasim.pro/blog/personal-slack-github-notifications-for-pull-requests-reviewers-how-to-set-up](https://rasim.pro/blog/personal-slack-github-notifications-for-pull-requests-reviewers-how-to-set-up/)

This post suggests using a GitHub feature named "[scheduled reminders](https://docs.github.com/en/organizations/organizing-members-into-teams/managing-scheduled-reminders-for-your-team)" for this. I do not think this is an appropriate solution for teams, as I mentioned in my original post.

**Comment by vkyt** [comment]
> > this works for me: [rasim.pro/blog/personal-slack-github-notifications-for-pull-requests-reviewers-how-to-set-up](https://rasim.pro/blog/personal-slack-github-notifications-for-pull-requests-reviewers-how-to-set-up/)
> 
> This post suggests using a GitHub feature named "[scheduled reminders](https://docs.github.com/en/organizations/organizing-members-into-teams/managing-scheduled-reminders-for-your-team)" for this. I do not think this is an appropriate solution for teams, as I mentioned in my original post.

Agree! Scheduled reminders does not work very well since it gives you a list of all open review requests.  It's hard to distribute the review among team members.

**Comment by echavemann** [comment]
+1

**Comment by timmkd** [comment]
+ 1

**Comment by timmkd** [comment]
+1

**Comment by willBampton** [comment]
+1

**Comment by lribeirocs** [comment]
+1

**Comment by thomas-pearson-mattr** [comment]
+1

**Comment by amy-hs** [comment]
+1

**Comment by kianjones9** [comment]
+1

**Comment by codebycarlos** [comment]
+1

**Comment by NaimKabir** [comment]
+1

**Comment by Skallwar** [comment]
+1

**Comment by cdelpone-beyond-finance** [comment]
+1

**Comment by viktor-urbanas-qatalog** [comment]
+1

**Comment by mgwedd** [comment]
+1

**Comment by ondikumana** [comment]
+1

**Comment by jen6** [comment]
+1

**Comment by mengtong-personal** [comment]
+1

**Comment by deadly-dagger** [comment]
+1

**Comment by jrgvf** [comment]
+1

**Comment by bhavik-jain-polestar** [comment]
+1

**Comment by ekscmorente** [comment]
+1

**Comment by tiffsaw** [comment]
+1

**Comment by olivwalsh** [comment]
+1

**Comment by srikotu** [comment]
+1

**Comment by justin-ki** [comment]
+1 do we know if this has been implemented yet?

**Comment by jamesHendersonAddepar** [comment]
Insane this isn't a feature. Additionally excluding by label, and excluding/including by author. I wish I could just add it myself lol

**Comment by amizberman-pave** [comment]
+1

**Comment by ijongkim** [comment]
+1

**Comment by pcova** [comment]
+1

**Comment by AntonioVentilii-DFINITY** [comment]
+1

**Comment by patrick-grow-therapy** [comment]
+1

**Comment by kartik-madhak** [comment]
+1

**Comment by liorkesten** [comment]
+1

**Comment by mjdavidson** [comment]
+1

**Comment by miseas** [comment]
+1

**Comment by myxomatozis** [comment]
+1

**Comment by ppseprus** [comment]
+1

**Comment by jojoms711** [comment]
+1

**Comment by prd-tan-truong** [comment]
+1

**Comment by beelarr** [comment]
+1

**Comment by VishnuKrishnathu** [comment]
+1

**Comment by yfngdu** [comment]
+1

**Comment by mchurichi** [comment]
+1

**Comment by gabriel-pineda** [comment]
If anyone is looking for an alternative, [pullnotifier](https://pullnotifier.com/) is a great option. We've been using it to get team-specific PR notifications and it's working exactly how we want it to.
